doc/fuzzing.md

```diff
@@ -139,7 +139,7 @@ You may also need to take care of giving the correct path for `clang` and
 Full configure that was tested on macOS Catalina with `brew` installed `llvm`:
 
 ```sh
-./configure --enable-fuzz --with-sanitizers=fuzzer,address,undefined CC=/usr/local/opt/llvm/bin/clang CXX=/usr/local/opt/llvm/bin/clang++ --disable-asm
+./configure --enable-fuzz --with-sanitizers=fuzzer,address,undefined --disable-asm CC=$(brew --prefix llvm)/bin/clang CXX=$(brew --prefix llvm)/bin/clang++
```

Comment:
Tested with brew-installed llvm 15.0.7, is there a workaround or open pull for the boost issues?

```
In file included from wallet/test/fuzz/notifications.cpp:11:
In file included from ./wallet/receive.h:11:
In file included from ./wallet/wallet.h:27:
./wallet/scriptpubkeyman.h:21:10: fatal error: 'boost/signals2/signal.hpp' file not found
#include <boost/signals2/signal.hpp>
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.

In file included from test/fuzz/block.cpp:14:
In file included from ./validation.h:29:
./txmempool.h:35:10: fatal error: 'boost/multi_index/hashed_index.hpp' file not found
#include <boost/multi_index/hashed_index.hpp>
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
```

Reply:
#27057. I don't think that's a blocker for fixing the docs.

Reply:
Nice.

Reply:
You can probably work around with --disable-wallet or so